src/scene/vcQueryNode.cpp

```diff
@@ -153,6 +153,11 @@ void vcQueryNode::HandleSceneExplorerUI(vcState *pProgramState, size_t *pItemID)
   int shape = m_shape;
   if (ImGui::Combo(udTempStr("%s##FilterShape%zu", vcString::Get("sceneFilterShape"), *pItemID), &shape, filterShapeNames, (int)udLengthOf(filterShapeNames)))
   {
+    // Change name if is still default
+    const char *defaultShapeNames[] = { vcString::Get("sceneExplorerFilterBoxDefaultName"), vcString::Get("sceneExplorerFilterCylinderDefaultName"), vcString::Get("sceneExplorerFilterSphereDefaultName") };
```

Comment:
You need to assert the size of this is equal to the number of possible shapes. A new shape being added won't trigger this code to be magically updated, and will eventually lead to a crash when someone inevitably changes to or from the new shape type.